README.md

```diff
@@ -116,26 +116,32 @@ class CountingDelegate: HTTPResponseDelegate {
 
     var count = 0
 
-    func didTransmitRequestBody() {
-        // this is executed when request is sent, called once
+    func didTransmitRequestPart(task: HTTPClient.Task<Response>, _ part: IOData) {
```

Comment:
@weissi @ianpartridge @tanner0101 do you think we need a callback when request head is sent as well? Also, what do you think about naming pattern (transmit/sent)?

Reply:
- I'd probably err on the side of 'more callbacks', @Lukasa can you think of a good use-case for a callback when the head has been sent?
- regarding naming transmit vs. send: I don't mind, think both are fine
- not sure about the `part: IOData` label, `part` is maybe a bit unclear? I would almost make it `_ part: IOData`, don't think the label adds a lot of value there given the type `IOData`? But this is really not super important :)

Reply:
but it's already `_ part: IOData` :)

Reply:
whoops :), sorry